tools/Fabric.Mcp.Tools.PublicApi/src/Resources/fabric-rest-api-specs/contents/mirroredDatabase/swagger.json

```diff
@@ -0,0 +1,630 @@
+{
```

Comment:
Do these swagger files get shipped with your MCP server? (I haven't seen all the files but I am curious how the MCP server uses them.)

Reply:
yes, we are shipping them as "resources" in this first PR.
We agreed with Jose to fetch them from an external source as a second phase in a separate PR.
Appreciate your approval of this PR if that's OK, as we try to get it checked-in for Fabric convention next week.